apps/hash-graph/libs/api/Cargo.toml

```diff
@@ -25,35 +25,35 @@ type-system = { workspace = true, features = ["utoipa"] }
 validation = { workspace = true, features = ["utoipa"] }
 codec = { workspace = true, optional = true }
 
-async-trait = "0.1.80"
+async-trait = "0.1.81"
 axum = "0.7.5"
 base64 = "0.22.1"
 bytes = "1.6.0"
 error-stack = { workspace = true }
 futures = { workspace = true }
-http-body-util = "0.1.1"
-hyper = "1.3.1"
-include_dir = "0.7.3"
+http-body-util = "0.1.2"
+hyper = "1.4.1"
+include_dir = "0.7.4"
 mime = "0.3.17"
-opentelemetry = "0.22.0"
-opentelemetry_sdk = { version = "0.22.1", features = ["rt-tokio"] }
+opentelemetry = "0.23.0"
+opentelemetry_sdk = { version = "0.23.0", features = ["rt-tokio"] }
 sentry = { version = "0.34.0", features = [
     "tracing",
     "tower",
     "tower-http",
 ], default-features = false }
-serde = { version = "1.0.200", features = ['derive'] }
-serde_json = "1.0.116"
+serde = { version = "1.0.204", features = ['derive'] }
+serde_json = "1.0.120"
 time = { workspace = true }
 tokio = { workspace = true, features = ["macros"], optional = true }
 tokio-postgres = { version = "0.7.10", default-features = false, optional = true }
 tokio-util = { workspace = true, features = ["io"], optional = true }
 tower = "0.4.13"
 tower-http = { version = "0.5.2", features = ["trace"] }
 tracing = { workspace = true }
-tracing-opentelemetry = "0.23.0"
-utoipa = "4.2.0"
-uuid = "1.8.0"
+tracing-opentelemetry = "0.24.0"
+utoipa = "4.2.3"
+uuid = "1.10.0"
```

Comment:
should we promote uuid, serde, serde_json to a workspace dependency while we're at it?